db/migrate/20200505095309_update_translations.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateTranslations < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :mobility_text_translations, :created_by, null: true, foreign_table_name: :users, index: true
+    add_column :mobility_text_translations, :is_approved, :boolean, null: false, default: true
+
+    add_reference :mobility_string_translations, :created_by, null: true, foreign_table_name: :users, index: true
+    add_column :mobility_string_translations, :is_approved, :boolean, null: false, default: true
```

Comment:
These add audit/review options for admins, tho it won't be fully implemented until we have Forms dynamically calling translated content. 

Adding them to both tables for now, bc I'd started out writing a materialized view to combine these, but then backed out and went with just adding a model for MobilityStringTranslations